commands/base.ts

```diff
@@ -23,7 +23,7 @@ export type InteractiveCommand<C extends Cmd> = (
 
 export interface CommandRunner extends CmdVisitor<ReturnValue> {
   executor: Executor;
-  args: Value[];
+  args: Array<Value | null>;
```

Comment:
This type is because the result of a runtime.interpret call can be a value or null. But the type of thing being evaluated, an interactive expression statement, will *always* return a value. Instead of making this nullable, I should assert that fact where the args array gets assembled.